modules/store.js

```diff
@@ -0,0 +1,30 @@
+export default class store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBook(book) {
+    const books = store.getBooks();
+
+    books.push(book);
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(authore) {
+    const books = store.getBooks();
+    books.forEach((book, index) => {
+      if (book.authore === authore) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/212907687-67da85cc-3fd4-407c-af97-c1d4e40b4452.png)

- [ ] You methods in the class are not arrow methods :fearful: Kindly convert them into arrow methods as i have done in the image above :arrow_up:

Reply:
- Well resolved @Shaheryar0054 👍🏽